### PR TITLE
Fix wording and puncutation of settings descriptions.

### PR DESCRIPTION
**Pull request by huantianad**
## Purpose
Settings menu descriptions were inconsistently punctuated, as well as awkwardly worded.

## Approach
Add periods to ends of all sentences, as well as reword slightly.

#### Open Questions and Pre-Merge TODOs
- [ ] Is "select" the correct word for game resolution, JVM memory, and JVM args?

### CODE REVIEW

File: src/common/modals/Settings/components/General.js [hunk 1]
```diff
@@ -348,7 +348,7 @@ const General = () => {
       <Content>
         <p>
           Select the number of concurrent downloads. If you have a slow
-          connection, select max 3
+          connection, select at max 3.
```
Comment (Eskaan) [inline src/common/modals/Settings/components/General.js]: I would prefer `at most` here

**Review by Eskaan — CHANGES_REQUESTED** [review]
All around a good pr, would definitely approve it if there weren't this little one.

File: src/common/modals/Settings/components/General.js (hunk 1 above)
Comment (huantianad) [reply src/common/modals/Settings/components/General.js]: Changed!

File: src/common/modals/Settings/components/General.js [hunk 2]
```diff
@@ -483,8 +483,8 @@ const General = () => {
       </Title>
       <Content>
         <p>
-          Deletes all the shared files between instances. Doing this will result
-          in the complete loss of the instances data
+          Deletes all the shared files between instances. Doing this will remove
+          ALL instance data.
```
Comment (huantianad) [inline src/common/modals/Settings/components/General.js:487]: Also changed up the wording here, not sure if there's a better way to word this

**Review by Eskaan — APPROVED** [review]

**Review by Eskaan — APPROVED** [review]
Oh didn’t know this feature existed on GH
But yes the changes are approved by myself. Good Idea to do that.

**Review by blarfoon — APPROVED** [review]

**Pull request merged**